sympy/assumptions/ask.py

```diff
@@ -706,8 +728,20 @@ def commutative(self):
         ``ask(Q.commutative(x))`` is true iff ``x`` commutes with any other
         object with respect to multiplication operation.
 
+        Examples
+        ========
+        >>> from sympy import ask, Q, MatrixSymbol, I
+        >>> X=MatrixSymbol('X', 2, 2)
+        >>> ask(Q.commutative(X))
+        >>>
```

Comment:
may be this can be removed.

Reply:
There are some issues with the output. I am working on it.